### PR TITLE
Salsa theming

### PR DESCRIPTION
**Pull request by 5adiyah**

### CODE REVIEW

**Review by unclegcb — COMMENTED** [review]
I'm pretty sure we don't want any of these to be in the repo:
```
.sass_cache/
Gemfile.lock
Rakefile
```
Probably need a more frontend-ready person to review here. @mortenson perhaps?

**Comment by mortenson** [comment]
@unclegcb Gemfile.lock is OK if you want to lock dependencies across builds, and Rakefile is required to build SASS. .sass_cache can be gitignored though.

File: sass/styles.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+@import 'https://fonts.googleapis.com/css?family=Archivo:400,400i,500,500i,600,600i,700,700i';
+@import 'https://fonts.googleapis.com/css?family=Merriweather:300,300i,400,400i,700,700i,900,900i';
+
+@import '../../../../themes/otfo_bene/sass/lib/bourbon/*';
+@import '../../../../themes/otfo_bene/sass/lib/neat/*';
+@import '../../../../themes/otfo_bene/sass/config/*';
```
Comment (mortenson) [inline sass/styles.scss]: This is specific to one client site right? Probably don't want it in the generic repo.

**Comment by unclegcb** [comment]
@5adiyah did we decide to move all of this out to the child theme? If so, can you get back on this branch and remove that stuff so I can merge in the changes to `SalsaEmbedStringFormatter.php`?

**Review by unclegcb — APPROVED** [review]

**Pull request merged**